UI/window-basic-main-outputs.cpp

```diff
@@ -2438,6 +2440,8 @@ bool AdvancedOutput::StartReplayBuffer()
 					     "RecRBSuffix");
 		rbTime = config_get_int(main->Config(), "AdvOut", "RecRBTime");
 		rbSize = config_get_int(main->Config(), "AdvOut", "RecRBSize");
+		bool overwriteIfExists = config_get_bool(
+			main->Config(), "Output", "OverwriteIfExists");
```

Comment:
```suggestion
```

This already occurs 12 lines above.

Reply:
Thank you, completely missed that.
Dunnae why it was there, it was completely unused.